### ISSUE TITLE
Release 4.5.1

### ISSUE THREAD

**Issue by thomaslevesque** [FakeItEasy/FakeItEasy#1318]
Can be labelled **ready** when all other issues on this milestone are closed.

- [x] Address all coding and style violations detected during a *Release* build (send a regular pull request which must be merged before continuing)
- [x] ensure completeness of draft [release notes](https://github.com/FakeItEasy/FakeItEasy/releases), including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
- [x] run `tools\prepare_release.cmd x.y.z` to:
    - change "vNext" names to appropriate x.y.z version for the next GitHub Release, milestone, and release checklist (this issue)
    - create a new draft GitHub Release, milestone, and release checklist
    - create release/x.y.z branch on upstream, branching from develop
    - submit a pull request to merge the release/x.y.z branch to master
- [x] merge the pull request, _which may be done by the PR creator, unlike a regular pull request_
- [x] tag master with the new version number and push to upstream. This will cause the build server to
    - push the nupkgs to [nuget.org](https://www.nuget.org/profiles/FakeItEasy) and
    - attach all nupkgs from the tagged build to the GitHub Release.
- [x] wait for NuGet package validation
- [x] de-list pre-release NuGet packages if present
- [x] publish the GitHub release
- [x] tweet, mentioning contributors, and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [x] post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- [ ] close this issue
- [ ] close the [milestone](https://github.com/FakeItEasy/FakeItEasy/milestones?state=open) for this release

**Comment by blairconrad** [comment]
#1328 is a big enough bug in the new-to-4.5.0 Analyzer functionality that I think we should release 4.5.1. I can get it out by Wednesday morning, if we'd like.

I'd also delist 4.5.0.

**Comment by thomaslevesque** [comment]
> #1328 is a big enough bug in the new-to-4.5.0 Analyzer functionality that I think we should release 4.5.1. I can get it out by Wednesday morning, if we'd like.

👍

**Comment by blairconrad** [comment]
Recanting on offer to delist 4.5.0. There's no big benefit - users picking up the package will go for 4.5.1 anyhow, and if upgrading, they'll probably look to see the value of 4.5.1 vs. 4.5.0.

**Comment by blairconrad** [comment]
Proposed release tweet:

```
Get FakeItEasy.Analyzer 4.5.1!
Fixed:
  RepeatedAssertionAnalyzer throws NullReferenceException when encountering an expression with no type
  Typo in docs
With thanks to Mertsch and @qayshp
https://github.com/FakeItEasy/FakeItEasy/releases/4.5.1
```

**Comment by thomaslevesque** [comment]
LGTM!

**Comment by blairconrad** [comment]
https://twitter.com/fakeiteasyfx/status/968809922258403329

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 4.5.1](https://github.com/FakeItEasy/FakeItEasy/releases/4.5.1).
